.env.example

```diff
@@ -41,7 +41,7 @@ MAX_FELTS_IN_CALLDATA=22500
 RETRY_TX_INTERVAL=10
 
 # Comma separated list of white listed pre EIP-155 transaction hashes
-WHITE_LISTED_EIP_155_TRANSACTION_HASHES=
+WHITE_LISTED_EIP_155_TRANSACTION_HASHES=0x0000000000000000000000000000000000000000
```

Comment:
I would leave these two variables blank in this file. If this is for testing purposes, you can use an approach similar to:

https://github.com/kkrt-labs/kakarot-rpc/blob/291a2c0ec59d8d75a25e61f24e418b19895137a8/tests/tests/eth_provider.rs#L753

to set a specific value to the env variable of your choice.

Reply:
fixed, thanks